Source/DafnyLanguageServer/Language/DafnyProgramVerifier.cs

```diff
@@ -142,6 +148,10 @@ private class ModelCapturingOutputPrinter : OutputPrinter {
 
       public void Inform(string s, TextWriter tw) {
         logger.LogInformation(s);
+        var match = Regex.Match(s, "^Verifying .+[.](?<name>[^.]+) ...$");
```

Comment:
The last three dots are any characters, can you escape them ?.
Also, this parsing of the verification message leaves aside the module and the class name when they are not default, which shouldn't be the case.

Reply:
Good point re. the `.`! 

I left out the module name on purpose; it's true that it's ambiguous if you have multiple methods in one file with the same name, but it keeps things short in the common case where you don't; the alternative makes for longer progress messages.  WDYT?

Reply:
Ok, let's keep it as such, since verification happens only in one file and usually they don't define different modules.